### ISSUE TITLE
Model works properly with NWT but failed with mf6 in flopy

### ISSUE THREAD

**Issue by hjia1005** [modflowpy/flopy#891]
Good day,

I am working on a big catchment area (150 rows and 141 columns,  cell size of 1000m). My plan was to take this opportunity to try mf6 with all the discretization packages in the same area and see the differences. At the beginning I had some converge issues and after consulting with Dr Richard Winston I changed some parameters in the ims package which helped the model to converge. However, the result head has never been right. Below is the description of my issue and my trials to solve the issue - but no success so far.

I tried many versions of mf6 discretization in the study area, including DISV with or without refinement along rivers (using gridgen to refine), also tried MF6 DIS. I used simple constant values (for K and rch), or ‘real value’ on each of the models. Nothing succeeded in mf6 in flopy. It always shows a normal termination but give me weird head results along that column or totally nonsense. 

I also ran the same above models in ModelMuse (but only with constant parameter values because the real data was only prepared in flopy), each of them runs properly and yields reasonable result. I also compared the model input files from flopy and from ModelMuse, still can see nothing wrong from the flopy side.

To be fair enough, I also ran the same structured grid model with NWT in flopy and it was fine – normal termination and reasonable head distribution. Now the problem seems existing with the use of mf6 in flopy? I plot everything in flopy and also export them to shapefiles to display in QGIS, everything was properly assigned.

I had some VB coding experience before, so would like to take this chance to get into python and flopy. I have been doing well, learning from my mistakes and corrections from Richard and from the flopy team. But this one looks like unsolvable. It’s quite frustrating. 

Attached please find the result head distribution from the mf6 models and nwt models for you to have an idea what I am talking about.

I am looking forward to your assistance on this. 

Best regards
hjia

![1_mf6](https://user-images.githubusercontent.com/33449394/82754793-b9402000-9dcf-11ea-9b53-e02323965c9f.png)
![2_mf6](https://user-images.githubusercontent.com/33449394/82754798-bb09e380-9dcf-11ea-84bc-042323445079.PNG)
![3_mf6](https://user-images.githubusercontent.com/33449394/82754800-bba27a00-9dcf-11ea-99df-dba50b11c343.png)
![4_mf6](https://user-images.githubusercontent.com/33449394/82754801-bc3b1080-9dcf-11ea-8552-1fc6bc5b99d9.PNG)
![5_nwt](https://user-images.githubusercontent.com/33449394/82754803-bcd3a700-9dcf-11ea-9299-7c0d93df011e.PNG)

Haili

**Comment by hjia1005** [comment]
Just got this with mf6-dis, the previous one was because I forgot to mask the 1e+30 values. Still working on disv to make it right.


![6_mf6dis](https://user-images.githubusercontent.com/33449394/82758682-45f6d800-9de8-11ea-9950-e8364b892d36.png)

**Comment by langevin-usgs** [comment]
@hjia1005, sorry to hear you are having problems.  Based on the information here, there is no way we can tell what is going on or if there is a bug.  If you can identify a bug and more precisely show the problem, then we can open this issue again.

**Comment by hjia1005** [comment]
Hi Chris,

Thanks for the reply. No. 4 in the above figures had no problem. It's only I forgot to mask the 1e+30 values in the plot.

Also I finally figured out the problem was actually with stress packages. Though values were applied properly from the technical aspect but the model cannot reach a good solution with those values. However what puzzeles me was, every model run had a normal termination and a balanced water budget (percent discrepancy less than 0.1%) while the result heads were weird. I was only using RCH and DRN packages. Later after I added the EVT package and increased recharge values the head distribution gets much better. 

So my lesson is, a normal termination with balanced water budget doesn't necessarily mean a converge?

Thank you again for taking time to look into my issue and thanks for the good work of your team.
